scripts/mocha-parallel.js

```diff
@@ -198,12 +214,25 @@ function runTests(taskConfigsFolder, run, options, cb) {
                 failures = reporter.failures;
 
             var duration = 0;
+            var catastrophicError = "";
             for (var i = 0; i < numPartitions; i++) {
                 var partition = partitions[i];
                 stats.passes += partition.passed;
                 stats.failures += partition.failed;
                 stats.tests += partition.tests;
                 duration += partition.duration;
+                if (partition.catastrophicError !== "") {
+                    //{"light":false,

```

Comment:
Can you add a comment to that effect? As it stands it just looks like commented-out code.